met-web/src/components/engagement/form/EngagementWidgets/Map/MapOptionCard.tsx

```diff
@@ -52,23 +54,16 @@ const MapOptionCard = () => {
             data-testid={`widget-drawer-option/${WidgetType.Map}`}
             elevation={1}
             sx={optionCardStyle}
-            onClick={() => createWidget()}
+            onClick={() => handleCreateWidget()}
         >
-            <If condition={creatingWidget}>
+            <If condition={isCreatingWidget}>
                 <Then>
                     <Grid container alignItems="center" justifyContent="center" direction="row" height="5.5em">
                         <CircularProgress color="inherit" />
                     </Grid>
                 </Then>
                 <Else>
-                    <Grid
-                        xs={12}
-                        container
-                        alignItems="center"
-                        justifyContent="flex-start"
-                        direction="row"
-                        columnSpacing={1}
-                    >
+                    <Grid container alignItems="center" justifyContent="flex-start" direction="row" columnSpacing={1}>
```

Comment:
these changes are intended rite?
xs={12} got removed..

Reply:
yes, it was causing an error in jest. And it was not effective anyway. xs prop only works if grid is item. Above wasn't item so xs had no effect and just caused warnings